modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/utils/APIService.js

```diff
@@ -46,8 +46,8 @@ function fetchWithError(url, options = {}) {
 	return fetch(url, options)
 		.then((response) => response.json())
 		.then((objectResponse) => {
-			if (objectResponse.error) {
-				throw objectResponse.error;
+			if (objectResponse?.error) {
```

Comment:
Could `objectResponse` not contain `error` but another thing that we may use? If we are throwing the whole object maybe could make sense to check only `objectResponse`